o/oauth2/oauth2_ubi8.4.sh

```diff
@@ -0,0 +1,65 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : oauth2
+# Version       : master
+# Source repo   : https://github.com/golang/oauth2
+# Tested on     : UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vikas Gupta <vikas.gupta8@ibm.com>
+# Language 	: GO
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#
+PACKAGE_NAME=oauth2
+PACKAGE_PATH=golang.org/x/oauth2
+PACKAGE_VERSION=${1:-master}
```

Comment:
it is probably okay to use master when there are no tags or releases but please put a comment to state that explicitly.

Reply:
Updated version.